listenbrainz/listenstore/redis_listenstore.py

```diff
@@ -40,3 +40,16 @@ def check_connection(self):
         except redis.exceptions.ConnectionError as e:
             self.log.error("Redis ping didn't work: {}".format(str(e)))
             raise
+
+    def store_latest_listens(self, listens):
+        try:
+            all_listens = self.redis.set('latest_listens', listens)
+        except redis.exceptions.ConnectionError as e:
+            self.log.error("The data can't be set due to: {}".format(str(e)))
+
+    def get_latest_listens(self):
+        """ This module is used to get the datas that has been set in the redis
+            Please use this function instead of store_latest_listens
+        """
+        return self.redis.get('latest_listens')
```

Comment:
you should slice this list to only return a maximum of 10 listens.

Reply:
Use `self.redis.getrange('latest_listens', 0, 9)` instead of this?

Reply:
I don't think that will work, to do data you'll have to make the key value an actual redis list when right now you're saving the dict directly.

Reply:
umm.. can I use array to pick ten first data?